### PR TITLE
Finalisation

### PR DESCRIPTION
**Pull request by frvanco**

### CODE REVIEW

File: src/components/QuizPage.jsx [hunk 1]
```diff
@@ -25,7 +25,12 @@ export default function QuizPage() {
   `;
 
   const Button = styled.div`
+<<<<<<< HEAD
+    display: flex;
+    justify-content: center;
+=======
     display: block;
+>>>>>>> b5b1c08bd5d8dbacfdf41e1941ba01e9a02acff3
```
Comment (LoicBrassart) [inline src/components/QuizPage.jsx]: Conflict trash

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Plz fix ! (No one read this PR before me -_-)

File: src/components/Header.jsx [hunk 2]
```diff
@@ -20,10 +17,9 @@ const Sh1 = styled.h1`
 const ButtonWrapper = styled.section`
   background-color: #222222;
   display: flex;
-
   justify-content: space-around;
-  padding-left: 20vw;
-  padding-right: 20vw;
+
```
Comment (Maenhout) [inline src/components/Header.jsx:21]: empty line

File: src/components/Header.jsx [hunk 3]
```diff
@@ -67,14 +63,13 @@ const Button = styled.button`
...
   align-items: center;
-
   text-decoration: none;
   color: inherit;
-  margin-top: 0.2em;
 `;
 
 const Myimg = styled.img`
   width: 1.5em;
+
```
Comment (Maenhout) [inline src/components/Header.jsx]: empty line

File: src/components/Header.jsx (hunk 2 above)
Comment (Maenhout) [inline src/components/Header.jsx:28]: empty line

File: src/components/Header.jsx [hunk 4]
```diff
@@ -83,14 +78,15 @@ const Myimg = styled.img`
...
-      </Sh1>
       <nav>
         <ButtonWrapper>
+          <Sh1>
+            <Slink to="/">
+              <Myimg src={logo} alt="Logo" />
+              <p>SUPER HEROES</p>
+            </Slink>
+          </Sh1>
+
```
Comment (Maenhout) [inline src/components/Header.jsx:88]: empty line

**Review by Maenhout — COMMENTED** [review]
Remove empty lines

**Review by Maenhout — APPROVED** [review]
pray to the lord that it does not bring desolation x)

**Review by LoicBrassart — APPROVED** [review]
No more Evil!

**Pull request merged**